scripts/simex

```diff
@@ -394,6 +395,13 @@ builds_remake_parser.set_defaults(cmd=do_builds_remake)
 # ---------------------------------------------------------------------------------------
 
 def do_develop(args):
+	# Initialize the cache.
+	if not os.path.isfile("./simex.cache"):
+		with open('./simex.cache', 'a+') as myfile:
```

Comment:
I would think it would be beneficial to have the `simex.cache` file hidden, thus naming it `.simex.cache`. I also think that the name of the cache file should be parameterized as a constant. This way, the naming of important files is not _hidden_ in code branches.